docs/download_files.md

```diff
@@ -75,4 +82,17 @@ Metadata pertaining to processing will also be available in this table and insid
 See the {ref}`Experiment metadata <sce_file_contents:experiment metadata>` section for more information on metadata columns that can be found in this file as well as inside the `SingleCellExperiment` object.
 
 For projects with bulk RNA-seq data, the `bulk_metadata.tsv` file will be included for project downloads. 
-This file will contain fields equivalent to those found in the `single_cell_metadata.tsv` related to processing the sample, but will not contain patient or disease specific metadata (e.g. `age`, `sex`, `diagnosis`, `subdiagnosis`, `tissue_location`, or `disease_timing`).
\ No newline at end of file
+This file will contain fields equivalent to those found in the `single_cell_metadata.tsv` related to processing the sample, but will not contain patient or disease specific metadata (e.g. `age`, `sex`, `diagnosis`, `subdiagnosis`, `tissue_location`, or `disease_timing`).
+
+## Download contents for samples with spatial transcriptomics
+
+For all spatial transcriptomics libraries, a `SCPCL000000_spatial` folder will be nested inside the corresponding sample folder in the download. 
+Inside that folder will be the following folders and files: 
+- A folder containing the [unfiltered counts files](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/matrices): `raw_feature_bc_matrix`,
+- A folder containing the [filtered counts files](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/matrices): `filtered_feature_bc_matrix`,
+- A folder containing the [spatial files](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/images): `spatial`,
```

Comment:
I don't like long links, but I think a bit of context for what the `spatial` folder contains would be useful here. I also moved the folder name to the front of the bullet point (a suggestion I am going to make for each one)

```suggestion
- A `spatial` folder containing [images and position information](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/images)
```